base/process_utils.h

```diff
@@ -0,0 +1,28 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
```

Comment:
I don't think this should be *_util. If we're going to create a generic utility it should be something similar to extensions `NativeProcessLauncher` and also this should be in base/process